terraform/gcp/projects/leap.tfvars

```diff
@@ -0,0 +1,75 @@
+prefix                 = "leap"
+project_id             = "leap-pangeo"
+core_node_machine_type = "n1-highmem-4"
+
+# No need for this to be a private cluster, public ones are cheaper
+enable_private_cluster = false
+
+# Multi-tenant cluster, network policy is required to enforce separation between hubs
+enable_network_policy  = true
+
+# Some hubs want a storage bucket, so we need to have config connector enabled
+config_connector_enabled = false
+
+# Setup a filestore for in-cluster NFS
+enable_filestore = true
+filestore_capacity_gb = 1024
+
+user_buckets = [
+  "pangeo-scratch"
```

Comment:
Didn't we establish elsewhere that this will create a bucket called Pangeo scratch, but JupyterHub is configured to look for a bucket called `<cluster_name>-<hub_name>` and so this bucket will never get connected to the hub?

Reply:
See slack message: https://2i2c.slack.com/archives/C028WU9PFBN/p1637755725001500?thread_ts=1637754043.001100&cid=C028WU9PFBN

And what JupyterHub is expecting: https://github.com/2i2c-org/infrastructure/blob/0e6006b87ad8a415c3b85c737046fcdc5c3731fa/hub-templates/basehub/values.yaml#L358

Reply:
@sgibson91 ah good catch. see https://github.com/2i2c-org/infrastructure/pull/1074/commits/d83461a216ef2b1d419bd62336a003680fbdf430.

Reply:
Perfect!